.github/workflows/release-emu.yml

```
@@ -0,0 +1,203 @@
+name: "[Release] emulators for trezor-user-env"
+
+on:
+  push:
+    tags:
+      - 'core/v*'
+      - 'legacy/v*'
+  workflow_dispatch:
+
+permissions:
+  id-token: write       # for fetching the OIDC token
+  contents: read        # for actions/checkout
+
+jobs:
+  get_models:
+    name: Get models
+    runs-on: ubuntu-latest
+    outputs:
+      models: ${{ steps.get_models.outputs.models }}
+      version: ${{ steps.get_models.outputs.version }}
+    steps:
+      - name: Get models from releases.json
+        id: get_models
+        run: |
+          if [[ $GITHUB_REF == refs/tags/core/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/core/v}
+            MODELS=$(jq -r --arg version "$VERSION" '.firmware[$version] | join(",")' common/releases.json)
+          elif [[ $GITHUB_REF == refs/tags/legacy/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/legacy/v}
+            MODELS=$(jq -r --arg version "$VERSION" '.firmware[$version] | join(",")' common/releases.json)
+          fi
+          echo "models=$MODELS" >> $GITHUB_OUTPUT
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+  core_emu:
+    if: startsWith(github.ref, 'refs/tags/core/v')
+    name: Build emu
+    needs: get_models
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        model: ${{ fromJson(needs.get_models.outputs.models) }}
+        coins: [universal]
+        type: [debuglink]
+        asan: [noasan]
+        exclude:
+          - type: normal
+            asan: asan
+    env:
+      TREZOR_MODEL: ${{ matrix.model == 'T2T1' && 'T' || matrix.model == 'T2B1' && 'R' || matrix.model }}
```

Should be
```suggestion
      TREZOR_MODEL: ${{ matrix.model }}
```
since #4477 was merged meanwhile

---

removed it